a2cps-cms/templates/fullwidth.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load cms_tags staticfiles %}
+
+{% block title %}{% page_attribute "page_title" %}{% endblock title %}
+
+{% block assets_font %}
+  {{ block.super }}
+{% endblock assets_font %}
+
+{% block content %}
+{% placeholder "content" %}
+{% endblock content %}
+
+{% block assets_custom %}
+  {{ block.super }}
+
+  <!-- TODO: GH-56 a.k.a. TACC/Core-CMS#245: Pending v1 styles -->
+  <link rel="stylesheet" href="{% static 'a2cps-cms/css/build/site.css' %}">
```

Comment:
For now, load the (empty) a2csp site stylesheet (which exists to show developer how to load custom styles).

Later—via GH-56 a.k.a. TACC/Core-CMS#245—load the v1 styles stylesheet.